internal/oidc/verifier_test.go

```
@@ -369,3 +379,410 @@ func makeJWTHeader(header map[string]any) string {
+
+	if _, err := mgr.getOrCreateVerifier(context.Background(), "cluster-a", clusterCfg); err == nil {
+		t.Fatal("expected error when both primary and fallback discovery fail")
+	}
+	if mgr.DegradedClusters()["cluster-a"] == "" {
+		t.Error("expected cluster-a to be degraded")
+	}
+}
+
+func doAndRead(t *testing.T, client *http.Client, url string) string {
+	t.Helper()
+	resp, err := client.Get(url)
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)